wcurl

```diff
@@ -213,10 +224,14 @@ exec_curl()
 
     NEXT_PARAMETER=""
     for url in ${URLS}; do
-        filename="$(get_url_filename "${url}")"
-        [ -z "${filename}" ] && filename=index.html
+        # If the user did not provide an output path, define one.
+        if [ "${HAS_USER_SET_OUTPUT}" = "false" ]; then
```

Comment:
`HAS_USER_SET_OUTPUT` is not really necessary.  We can check directly if `OUTPUT_PATH` is empty, and then proceed.

Reply:
Ah, you're right, I forgot about this case, thanks reminding me of it.